src/progress_bar.rs

```diff
@@ -609,12 +644,14 @@ impl MultiProgress {
     pub fn clear(&self) -> io::Result<()> {
         let mut state = self.state.write().unwrap();
         let move_cursor = state.move_cursor;
+        let alignment = state.alignment;
         state.draw_target.apply_draw_state(ProgressDrawState {
             lines: vec![],
             orphan_lines: 0,
             finished: true,
             force_draw: true,
             move_cursor,
+            alignment,
```

Comment:
Could we just assign `state.alignment` directly?

Reply:
Unfortunately, this wouldn't compile: _error[E0502]: cannot borrow `state` as immutable because it is also borrowed as mutable_
```rust
        state.draw_target.apply_draw_state(ProgressDrawState {
            lines: vec![],
            orphan_lines: 0,
            finished: true,
            force_draw: true,
            move_cursor: state.move_cursor,
            alignment: state.alignment,
        })?;
```

Reply:
Ah, that's what I was afraid of.